indexers/keyvalue/RedisDBIndexer/README.md

```diff
@@ -1,3 +1,47 @@
 # RedisDBIndexer
 
-Use Redis as a key-value indexer 
\ No newline at end of file
+Key-value indexer wrapper around the Redis database. Redis is an in-memory key-value storage system. You can read more about it here: https://redis.io/
+
+**NOTE** You will need to manually delete the keys in Redis if you choose to delete a workspace. 
+
+## Snippets:
+
+Initialise RedisDBIndexer:
+
+`RedisDBIndexer(hostname=host_ip, port=host_port, db=db_int)`
+
+Users can use Pod images in several ways:
+
+- Run with Docker (`docker run`)
+  
+  ```bash
+    docker run jinahub/pod.indexer.redisdbindexer:MODULE_VERSION-JINA_VERSION --port-in 55555 --port-out 55556
+    ```
+
+- Flow API
+  
+  ```python
+    from jina.flow import Flow
+    f = (Flow()
+        .add(name='my-indexer', uses='docker://jinahub/pod.indexer.redisdbindexer:MODULE_VERSION-JINA_VERSION', port_in=55555, port_out=55556)
+    ```
+
+- Jina CLI
+  
+  ```bash
+  jina pod --uses jinahub/pod.indexer.redisdbindexer:MODULE_VERSION-JINA_VERSION --port-in 55555 --port-out 55556
```

Comment:
```suggestion
  jina pod --uses docker://jinahub/pod.indexer.redisdbindexer:MODULE_VERSION-JINA_VERSION --port-in 55555 --port-out 55556
```

Reply:
`--port-in 55555 --port-out 55556` are unnecessary.